apps/fz_vpn/lib/fz_vpn/server.ex

```diff
@@ -46,21 +45,28 @@ defmodule FzVpn.Server do
     {:ok, new_config}
   end
 
+  def iface_name do
```

Comment:
defp?

Reply:
The `StatsPushService` module uses it to get the iface name to dump stats from, so I left it public.